packages/sheets-thread-comment/src/controllers/menu.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { IMenuItem, IShortcutItem } from '@univerjs/ui';
+import { getMenuHiddenObservable, KeyCode, MenuItemType, MenuPosition, MetaKeys } from '@univerjs/ui';
+import type { IAccessor } from '@wendellhu/redi';
+import { ToggleSheetCommentPanelOperation } from '@univerjs/thread-comment-ui';
+import { UniverInstanceType } from '@univerjs/core';
+import { whenSheetEditorFocused } from '@univerjs/sheets-ui';
+import { ShowAddSheetCommentModalOperation } from '../commands/operations/comment.operation';
+import { COMMENT_SINGLE_ICON } from '../types/const';
+
+export const threadCommentMenu = (accessor: IAccessor) => {
```

Comment:
The names of menu factories should adhere to our naming conversion. Refer to menu factories in `sheets` package.